solutions/cf-585D.mdx

```diff
@@ -0,0 +1,125 @@
+---
+id: cf-585D
+source: CF
+title: "Lizard Era: Beginning"
+author: Albert Ye
+---
+
+[Official Analysis](https://codeforces.com/blog/entry/20898)
+
+We notice that the size of the array is 25. A naive brute-force would take at least $\mathcal O(3^n)$ which is too slow
+because $n = 25$ and $3^25 = 8.47 \cdot 10^10 > 5 \cdot 10^9$. 
```

Comment:
```suggestion
because $n = 25$ and $3^25 = 8.47 \cdot 10^10 > 5 \cdot 10^9$. 
```

I would recommend using https://aops.com/texer to render your latex first to see how it comes out before pasting it here. Time for some heavy change reqs!

Reply:
ok my bad !!!!!!!!!